### DISCUSSION TITLE
Can't I install quantlib:x64-windows?

### DISCUSSION THREAD

**Discussion by KRJackLee** [microsoft/vcpkg#28826] [Q&A]
I tried to install quantlib with triplet x64-windows. Here's the result:
```
Computing installation plan...
quantlib[core]:x64-windows is only supported on '!(windows & !staticcrt)'
```
I think it relates to `ports/quantlib/vcpkg.json`, and I checked github repo to confirm that my `vcpkg.json` is latest:
```
{
  "name": "quantlib",
  "version": "1.28",
  "description": "The QuantLib C++ library",
  "homepage": "https://www.quantlib.org/",
  "license": "BSD-3-Clause",
  "supports": "!(windows & !staticcrt)",
  "dependencies": [
    "boost-accumulators",
    "boost-algorithm",
    "boost-any",
    "boost-array",
    "boost-assert",
    "boost-assign",
    "boost-atomic",
    "boost-bind",
    "boost-config",
    "boost-core",
    "boost-date-time",
    "boost-dynamic-bitset",
    "boost-format",
    "boost-function",
    "boost-functional",
    "boost-iterator",
    "boost-lexical-cast",
    "boost-math",
    "boost-multi-array",
    "boost-multiprecision",
    "boost-optional",
    "boost-preprocessor",
    "boost-random",
    "boost-serialization",
    "boost-signals2",
    "boost-smart-ptr",
    "boost-thread",
    "boost-tuple",
    "boost-type-traits",
    "boost-ublas",
    "boost-unordered",
    "boost-utility",
    {
      "name": "vcpkg-cmake",
      "host": true
    },
    {
      "name": "vcpkg-cmake-config",
      "host": true
    }
  ]
}
```
But at `https://vcpkg.io/en/packages.html` it states that `quantlib` is "Compatible with x64-windows". Why could it be the case? As a old-school package, I doubted that it missed a build with msvc or visual-studio or the like. By the way, x86-windows failed to install also. How could I install `quantlib:x64-windows` with vcpkg then?

**Comment by Osyotr** [comment]
This was changed in https://github.com/microsoft/vcpkg/pull/25191
Relevant code in quantlib: https://github.com/lballabio/QuantLib/blob/7bfe85c671a8fc4c5abe133032969c34d2e59963/cmake/Platform.cmake#L5-L14
Basically, it picks dynamic CRT only if `BUILD_SHARED_LIBS` is defined, but if `BUILD_SHARED_LIBS` then we get error message few lines below. Please, open an issue in upstream repository.
